<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smart_calc</class>
 <widget class="QWidget" name="smart_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>smart_calc</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="6">
    <widget class="QPushButton" name="opr_cos">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>cos(</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="opr_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="Equals">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #FFA07A;
}

QPushButton:pressed {
	BackGround-color: #B22222;
}

</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="opr_sqrt">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>sqrt(</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_Xmax">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Xmax =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_Xmax">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="opr_0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="opr_sin">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>sin(</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="opr_point">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="opr_atan">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>atan(</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="lineEdit_Ymax">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_Ymax">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Ymax =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_Xmin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="AllClear">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #FFE4B5;
}

QPushButton:pressed {
	BackGround-color: #DEB887;
}

</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="xButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #B8B3FF;
}

QPushButton:pressed {
	BackGround-color: #009900;
}

</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="opr_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QPushButton" name="opr_acos">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>acos(</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="opr_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="opr_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QCustomPlot" name="widget_plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="opr_log">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>log(</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="opr_bracket_left">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="opr_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="opr_sub">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="opr_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="opr_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="Clear">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #FFE4B5;
}

QPushButton:pressed {
	BackGround-color: #DEB887;
}

</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="opr_sum">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="opr_ln">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>ln(</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="opr_9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="opr_sqr">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_Ymin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="opr_mul">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="Sign">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="opr_div">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QPushButton" name="opr_asin">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>asin(</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="opr_bracket_right">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Xmin">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Xmin =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="opr_tan">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #DCDCDC;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>tan(</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="opr_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="opr_mod">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	BackGround-color: #D3D3D3;
}

QPushButton:pressed {
	BackGround-color: #A9A9A9;
}

</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_Ymin">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Ymin =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="7">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionEngineer">
   <property name="text">
    <string>Engineer</string>
   </property>
  </action>
  <action name="actionCredit">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
